tests/execution/test-union-fieldtype.md

```diff
@@ -0,0 +1,192 @@
+# Test union type resolve
+
+```graphql @config
+schema @server @upstream(baseURL: "http://jsonplaceholder.typicode.com") {
+  query: Query
+}
+
+union FooBar @discriminate(field: "type") = Bar | Foo
```

Comment:
Need a validation that the field described in the directive should be required in the variants.

Reply:
changed logic to use discriminate to field resolver level